README.md

```diff
@@ -26,9 +26,9 @@ The specifications come in several layers:
      is an Internet Draft, and a [draft of the next
      snapshot](https://wicg.github.io/webpackage/draft-yasskin-httpbis-origin-signed-exchanges-impl.html)
      is in this repository.
-1. [Bundled HTTP exchanges](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html)
+1. [Bundled HTTP exchanges (also called Web Bundles)](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html)
```

Comment:
We could just switch the primary name here.

```suggestion
1. [Web Bundles (previously called Bundled HTTP exchanges)](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html)
```

And I could change the title of the IETF draft.